templates/jlink/jlink-builder/jlink-builder-template.yaml

```diff
@@ -29,6 +29,7 @@ objects:
       dockerfile: |
         FROM -
         USER 0
+        RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y java-${JDK_VERSION}-openjdk-jmods
         RUN mkdir -p /mnt/jrootfs
         RUN microdnf install --installroot /mnt/jrootfs --releasever 9 --setopt install_weak_deps=0 --nodocs -y java-${JDK_VERSION}-openjdk-jmods \
```

Comment:
this line is still installing the jmods RPM into the jrootfs directory

Reply:
Oops sorry :(  copy paste mistake.